src/main/java/frc/robot/constants/DriveConstants.java

```diff
@@ -1,9 +1,156 @@
 package frc.robot.constants;
 
-public class DriveConstants {
+import edu.wpi.first.math.util.Units;
 
-  public static final int kLeftMotor1 = 1;
-  public static final int kLeftMotor2 = 2;
-  public static final int kRightMotor1 = 3;
-  public static final int kRightMotor2 = 4;
+/**
+ * Container class for drive constants.
+ */
+public class DriveConstants {
+  
+  public static final double kWheelRadius = Units.inchesToMeters(2);
+  
+  public static final double kTrackWidth = Units.inchesToMeters(20.75);//22.75 swerve bot, 20.75 comp bot
+  public static final double kDriveGearRatio = 6.75;
+  public static final double kSteerGearRatio = 12.8;
+  
+  public static final double kMaxSpeed = FalconConstants.kMaxRpm / 60.0 / kDriveGearRatio * kWheelRadius * 2 * Math.PI;
+  
+  public static final double kMaxAngularSpeed = FalconConstants.kMaxRpm / 60.0 / kSteerGearRatio * 2 * Math.PI; // 8.3 rot/s
```

Comment:
This is not the max angular speed of the robot. For that, just take the max speed of the robot and divide by the radius (v = w/r).